browser/brave_ads/android/java/org/chromium/chrome/browser/dialogs/BraveAdsSignupDialog.java

```diff
@@ -142,9 +144,12 @@ public static void showExistingUserDialog(Context context) {
             public void onClick(DialogInterface dialog, int which) {
                 // Enable ads
                 neverShowOnboardingDialogAgain();
-                BraveActivity braveActivity = BraveActivity.getBraveActivity();
-                if (braveActivity != null) {
-                    braveActivity.openRewardsPanel();
+                try {
+                    BraveActivity braveActivity = BraveActivity.getBraveActivity();
+                    if (braveActivity != null) {
+                        braveActivity.openRewardsPanel();
+                    }
+                } catch (Exception e) { // sujit
```

Comment:
Should be ActivityNotFoundException. Also not sure what this comment implies.

Reply:
updated